_data/identity.yml

```diff
@@ -100,8 +100,9 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+      phone: Yes
       hardware: Yes
-      doc: http://www.okta.com/what-we-do/multifactor-authentication.html
+      doc: https://www.okta.com/products/adaptive-multi-factor-authentication/
```

Comment:
`doc:` url should be not a marketing product page with what okta is selling, but a link to a documentation how a user can enable 2fa for their okta account.